jmix-core/core/src/main/java/io/jmix/core/impl/repository/query/JmixParameters.java

```diff
@@ -17,27 +17,31 @@
 package io.jmix.core.impl.repository.query;
 
 import io.jmix.core.FetchPlan;
-import org.springframework.core.MethodParameter;
+import io.jmix.core.repository.JmixDataRepositoryContext;
 import org.springframework.data.repository.query.Parameters;
+import org.springframework.data.repository.query.ParametersSource;
 
 import java.lang.reflect.Method;
 import java.util.List;
+
 /**
  * {@link Parameters} extension required to support {@link io.jmix.core.FetchPlan} special parameter.
  */
 public class JmixParameters extends Parameters<JmixParameters, JmixParameter> {
 
     private final int fetchPlanIndex;
+    private final int jmixArgsIndex;
```

Comment:
The field name and related methods should probably be renamed (jmixDataRepositoryContextIndex, not jmixArgIndex)